fe1-web/features/events/screens/Organizer.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
```

Comment:
shouldn't this file go into a `features/lao/screens` folder ?

Reply:
I thought of it as a "list of all events", that's why I put it there. But since we're aiming to remove this `features/events`, I guess this screen fits better in `features/lao/screens`